torch/utils/serialization/read_lua_file.py

```diff
@@ -366,6 +366,12 @@ def storage_to_size(obj, *attrs):
         setattr(obj, attr, torch.Size(value))
 
 
+@registry_addon
+def storage_to_FP(obj, attr):
+    value = getattr(obj, attr)
+    setattr(obj, attr, value)
```

Comment:
sorry, but what is this function supposed to do? looks like a no-op to me

Reply:
sorry for that, yes just wanted to add PixelShuffle to the reader registry, now we just used the ensure_attr function to do this